cheatsheets/Tower.rb

```diff
@@ -0,0 +1,189 @@
+cheatsheet do
+  title 'Tower'
+  docset_file_name 'Tower'
+  keyword 'tower'
+  source_url 'http://cheat.kapeli.com'
+  introduction 'Keyboard shortcuts for **Tower**, the most powerful Git client for Mac and Windows - visit [www.git-tower.com](https://www.git-tower.com) for more.'
+
+  category do
+    id 'General'
+    entry do
+      command 'CMD+N'
+      name 'Open new **window**'
+    end
+    entry do
+      command 'CMD+SHIFT+O'
+      name 'Show **Quick Open** dialog'
+    end
+    entry do
+      command 'CTRL+SHIFT+A'
+      name 'Show **Quick Actions** dialog'
+    end
+    entry do
+      command 'CTRL+SHIFT+D'
+      name 'Open external **diff tool** for current view'
+    end
+    entry do
+      command 'CTRL+CTRL+0'
```

Comment:
Is the double `CTRL` correct here or a mistake?

Reply:
That should indeed be "CMD+CTRL+0" for the "Activity window" and also "CMD+CTRL+F" for "Full screen" (right below). Thanks for double-checking that!